ext/civiimport/Civi/Api4/Event/Subscriber/ImportSubscriber.php

```diff
@@ -62,8 +62,8 @@ public static function on_civi_api4_entityTypes(GenericHookEvent $event): void {
       /** @noinspection PhpUndefinedFieldInspection */
       $event->entities['Import_' . $userJobID] = [
         'name' => 'Import_' . $userJobID,
-        'title' => ts('Import Data') . ' ' . $userJobID . (empty($table['created_by']) ? '' : '(' . $table['created_by'] . ')'),
-        'title_plural' => ts('Import Data') . ' ' . $userJobID,
+        'title' => $table['title'] . ' ' . $table['description'],
+        'title_plural' => $table['title'],
         'description' => ts('Import Job temporary data'),
```

Comment:
What's stored in `$table['description']` and why not put it here?

Reply:
It's created date and created by. I think it would make the title_plural, which is the one you see when you select the entity in SK for example, way too long. Could also make them both just title, I don't really know why the non-plural title contains created by.

Reply:
Yea I'd say let's put it as the description here.